project_stage_last_update_date/models/project_project.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 Tecnativa - Carolina Fernandez
+# License LGPLv3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html).
+from datetime import datetime
+
+from odoo import fields, models
+
+
+class Project(models.Model):
+    _inherit = "project.project"
+
+    last_update_date_stage = fields.Datetime(copy=False, readonly=True, tracking=True)
```

Comment:
It shouldn't be tracked IMO. You already have the tracking in the stages, isn't it? So looking at that date you have the data.

Reply:
And a more descriptive field name is `stage_last_update_date`. Why reversing it?